src/components/TextInput/TextInput.tsx

```diff
@@ -33,6 +33,7 @@ const TextInput: React.FC<TextInputProps> = (props) => {
           {...(rest as any)}
           autoComplete={autoComplete}
           ref={(ref) => getRef(ref)}
+          type={isNumeric ? 'number' : 'text'}
           onChange={(e) => {
```

Comment:
due to type="number" now its showing up down arraow in all texbox
![Screenshot 2022-09-06 at 12-14-25 Pangolin](https://user-images.githubusercontent.com/6604146/188565154-70eb9b88-c975-4ccc-a76b-a4f6938ea342.png)

Reply:
I've fixed it.